lib/CL/pocl_mem_management.c

```diff
@@ -1105,3 +1105,41 @@ pocl_cpu_get_ptr (struct pocl_argument *arg, unsigned global_mem_id)
   ptr += arg->offset;
   return (void *)ptr;
 }
+
+void
```

Comment:
For MSVC toolchain you'll need add `POCL_EXPORT` here too.

Reply:
All CIs passed. Why didn't MSVC scream here?

Reply:
Probably there is no callers from outside libpocl right now so that's why you won't hear any screaming.

Reply:
Right. Actually I'd say the POCL_EXPORT is extra here as well since we currently call from libpocl.so only and not from the driver plugins.